src/notation/internal/notationinteraction.h

```diff
@@ -249,6 +249,7 @@ class NotationInteraction : public INotationInteraction, public async::Asyncable
         std::vector<Element*> elements;
         std::vector<std::unique_ptr<Ms::ElementGroup> > dragGroups;
         void reset();
+        DragMode mode;
```

Comment:
It would be nice to initialize with a default value

Reply:
I don't mind adding it but then it might look odd, all the other fields in the struct assume they're auto initialised with the default value...as it happens the field is meaningless until some of the other fields get set too.
But I do note that I added this field *after* a method declaration (`void reset()`), it's probably better to move it above it along with the others?

Reply:
Other fields have (or should have) constructors, so they have default initialization.
Enums have no constructors and are accordingly initialized with a random value.
The code is changing all the time ... and, accordingly, a situation may accidentally arise when they will use it before the assignment of any value, and since the initialized value is random, this can lead to hard-to-reproduce errors.
Therefore, we always strive to initialize all fields with default values, for example, you can add an undefined value to the enum and initialize it.

Reply:
Sorry, keep having to mentally adjust from C#-land, anyway seems there's a new syntax for declaring initializer values, which is used elsewhere so have done that (thought I'm not sure how it differs from just using = ).